nixos/modules/services/monitoring/riemann-tools.nix

```diff
@@ -34,8 +37,24 @@ in {
           Address of the host riemann node. Defaults to localhost.
         '';
       };
+      attributes = mkOption {
```

Comment:
I feel like this doesn't add a lot of value over `commandSwitches` (which I would prefer be called `extraArgs`) -- can we scrap it? Keeping NixOS modules small and simple is good for evaluation time and maintainability :)